cekit/templates/template.jinja

```diff
@@ -126,6 +126,10 @@ WORKDIR {{ run['workdir'] }}
 ENTRYPOINT {{ helper.cmd(run['entrypoint']) }}
 {% endif %}
 
+{% if addhelp %}
```

Comment:
i would like to move this at the end as I would like to keep ENTRYPOINT and CMD  together.

Reply:
np - will fix with my next push